src/max_batch_size.md

```diff
@@ -0,0 +1,125 @@
+---
```

Comment:
I'd suggest updating the "Server Guide" page and add links to think page for each resource method. Say something like "The batch size of requests for a ____ method can be limited and validated by configuring a max batch size" with a link to this page

Reply:
Good suggestion, added.

Reply:
Good suggestion, added